build-tools/tasks/test-utils.js

```diff
@@ -57,12 +105,22 @@ function generateIndexFileContent(testUtilType, testUtilMetaData) {
     ...testUtilMetaData.map(({ componentName }) => {
       const wrapperName = toWrapper(componentName);
       // language=TypeScript
-      return `ElementWrapper.prototype.find${componentName} = function(selector) {
-          const rootSelector = \`.$\{${wrapperName}.rootSelector}\`;
-          // casting to 'any' is needed to avoid this issue with generics
-          // https://github.com/microsoft/TypeScript/issues/29132
-          return (this as any).findComponent(selector ? appendSelector(selector, rootSelector) : rootSelector, ${wrapperName});
-      };`;
+      return `
+          ElementWrapper.prototype.find${componentName} = function(selector) {
+            const rootSelector = \`.$\{${wrapperName}.rootSelector}\`;
+            return this.findComponent(selector ? appendSelector(selector, rootSelector) : rootSelector, ${wrapperName});
+          };
+
+          ElementWrapper.prototype.findAll${pluralize(componentName)} = function(selector) {
```

Comment:
How does it work with the Cards, Steps, Tabs components? Does it print `findAllCards` (which is confusing if it is an array of `CardWrapper` or `CardsWrapper`).

If you want my opinion, any auto-plurlalization is a dead end which does not scale. The method should include a the canonical name  in its original form, for example `findCardsAll`

Reply:
> How does it work with the Cards, Steps, Tabs components?

Some output
```tsx
pluralize('cards')  // cards
pluralize('steps')  // steps
pluralize('tabs')  // tabs
pluralize('canvas')  // canvases
plurazlie('plus')  // pluses
pluralize('watch')  // watches
```

> If you want my opinion, any auto-plurlalization is a dead end which does not scale.

How far do we want to scale this? We have 82 components in the meantime and it covers all of those as well as the components in the future to come. Also looking at [its source code](https://github.com/plurals/pluralize/blob/master/pluralize.js#L269), they have a quite elaborate grammar ruleset for pluralization which should be more than enough for our use case.

> The method should include the canonical name in its original form, for example findCardsAll

This is a good suggestion too. `findAllXX` is closer to RTL's API.